client/src/components/topics-search.jsx

```diff
@@ -0,0 +1,196 @@
+import React, { Component } from 'react';
+import { ApiService } from '../services';
+import { navigate, Link } from '@reach/router';
+import { Dropdown, DropdownButton } from 'react-bootstrap';
+
+class TopicSearch extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      _id: '',
+      name: '',
+      searchTerm: '',
+      searchParameter: '',
+      dropDownValue: 'Topic',
+      searchComplete: false,
+      data: [],
+    };
+  }
+
+  async componentDidMount() {
+    await ApiService.get('topics').then(
```

Comment:
I ended up not using this component. Because there is no longer a master list of topics and search terms (as there is with publications), we don't need to search for a topic. Users can just add all the topics they want, and if the topic is already in their list, they get the 409 error. Do you think I should be more specific on the error, with a separate component that says that "'X' is already in your list of search terms:?

Reply:
For now, handle the same as if it were added. We can enhance later if needed.